### DISCUSSION TITLE
Generating new chunks on downloaded worlds

### DISCUSSION THREAD

**Discussion by imjohne** [mircokroon/minecraft-world-downloader#546] [Q&A]
So when will this start to work again cause I know that it used to work but has been broken for a long time now

**Comment by AbdiAbdiAbdi** [comment]
agree. i also wanna keep playing on these downloaded maps. maybe there's a way to edit it. the only thing i have in my mind about it is to just copy the map you downloaded with the "world edit" mod and paste it into a new generated world.
Edit: using world edit actually works. here's a tutorial: https://www.youtube.com/watch?v=KGcg3qqupNk
